synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp

```diff
@@ -122,6 +125,14 @@ ffmpeg_trgt::~ffmpeg_trgt()
 	file=NULL;
 	delete [] buffer;
 	delete [] color_buffer;
+
+	// Remove temporary sound file
+	struct stat buf;
+	if (g_stat(sound_filename.c_str(), &buf) != -1) {
+		if( g_remove(sound_filename.c_str()) != 0 ) {
+			synfig::warning("Error deleting temporary sound file.");
```

Comment:
I suggest adding the filename to the message so that the user knows which one they are trying to delete.

Reply:
Done.